share/commands/agent-run

```diff
@@ -83,4 +90,8 @@ if [ -e /opt/rudder/etc/disable-agent ]; then
   exit 1
 fi
 
+if [ "${UPDATE}" = "true" ]; then
```

Comment:
I think you should add raw and multiline options to UPDATE_OPTIONS since they are formatting options too